Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -1481,6 +1481,40 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     if shouldSkip(node) {
       return .skipChildren
     }
+
+    if let modifiers = node.modifiers, modifiers.hasError {
+      for modifier in modifiers {
+        if let detail = modifier.detail, detail.detail != .keyword(.set) {
```

Comment:
Why do we need to check for `detail.detail != .keyword(.set)` here?

---

You could decrease the nesting level a bit by doing

```swift
guard let detail = modifier.detail, detail.detail != .keyword(.set) else {
  continue
}
```

instead.